hexrd/resources/tardis_reference_config.yml

```diff
@@ -8,6 +8,7 @@ beam:
     polar_angle: 90.0
 detectors:
   IMAGE-PLATE-2:
+    detector_type: 'planar'
```

Comment:
Since most detectors are `planar`, I think it's fine if we just leave out the `detector_type` if it is planar (as it will default to being `planar`). But we can still put it here if it helps clarify that it is planar.